packages/rum-core/src/common/bootstrap.js

```diff
@@ -23,24 +23,17 @@
  *
  */
 
-import { isPlatformSupported } from './common/utils'
-import { patchAll } from './common/patching'
-import { bootstrapMetrics } from './performance-monitoring/metrics'
+import { isPlatformSupported, isBrowser } from './utils'
+import { patchAll } from './patching'
+import { bootstrapMetrics } from '../performance-monitoring/metrics'
 
-let alreadyBootstrap = false
```

Comment:
This logic is redundant as its handled while creating `apmBase` singleton instance.